docs/en/latest/architecture-design/apisix.md

```diff
@@ -49,3 +49,70 @@ The diagram below shows how APISIX handles an incoming request and applies corre
 The chart below shows the order in which different types of Plugin are applied to a request:
 
 ![flow-plugin-internal](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/flow-plugin-internal.png)
+
+## Examples of route matching
+
+1. Host + URI: This criterion matches requests based on the combination of the host and URI. For example, you could use this to match all requests to https://example.com/api.
```

Comment:
> match all requests to https://example.com/api ?

Or `https://example.com/foo/bar`?

Reply:
I have written here by taking some random examples in all three cases. Please tell me about this.